pincer/objects/message/file.py

```diff
@@ -28,8 +31,58 @@
     PILLOW_IMPORT = False
 
 
+def create_form(
+    json_payload: Dict[Any], files: List[File]
+) -> Tuple[str, Union[Payload, Dict]]:
+    """
+    Creates a aiohttp payload from an array of File objects.
+
+    json_payload : Dict[Any]
+        The json part of the request
+    files : List[`~pincer.objects.message.file.File`]
+        A list of files to be used in the request.
+
+    Returns
+    -------
+    `aiohttp.Payload`
+        The payload to be send in an HTTP request.
+    """
+    form = FormData()
+    form.add_field("payload_json", dumps(json_payload))
+
+    for file in files:
+        if not file.filename:
+            raise ImageEncodingError(
+                "A filename is required for uploading attachments"
+            )
+
+        form.add_field("file", file.content, filename=file.filename)
+
+    payload = form()
+    return payload.headers["Content-Type"], payload
+
+
+def _get_file_extension(filename: str) -> Optional[str]:
+    """
+    Returns the file extension from a str if it exists, otherwise
+    return :data:`None`.
+
+    filename : str
+        The filename
+
+    Returns
+    -------
+    Optional[str]
+        The file extension or :data:`None`
+    """
+    path = os.path.splitext(filename)
+    if len(path) >= 2:
+        return path[1][1:]
+    return None
```

Comment:
`return None` is uneeded

Reply:
You have to explicitly state `return None` if there are any returns according to pep8
> Be consistent in return statements. Either all return statements in a function should return an expression, or none of them should. If any return statement returns an expression, any return statements where no value is returned should explicitly state this as return None, and an explicit return statement should be present at the end of the function (if reachable)

https://www.python.org/dev/peps/pep-0008/